src/cmd-prune-pkgcache

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+print_help() {
+    cat 1>&2 <<'EOF'
+Usage: coreos-assembler prune-pkgcache --help
+       coreos-assembler prune-pkgcache
+
+  Prune refs packages from the pkgcache, which are not present in the builds and modified over 30 days.
+EOF
+}
+
+rc=0
+options=$(getopt --options h --longoptions help -- "$@") || rc=$?
+[ $rc -eq 0 ] || {
+    print_help
+    exit 1
+}
+eval set -- "$options"
+while true; do
+    case "$1" in
+        -h | --help)
+            print_help
+            exit 0
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            fatal "$0: unrecognized option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+if [ $# -ne 0 ]; then
+    print_help
+    fatal "ERROR: Too many arguments"
+    exit 1
+fi
+
+#Convert all the builds pkg names to the naming convention
+bdspkgstr=$(jq '."rpmostree.rpmdb.pkglist"' "$(find /srv/builds/ -name "commitmeta.json")" \
```

Comment:
It might be simplest actually to prune all cached packages not referenced by the *latest* commit, i.e. we just look at `builds/latest/$(arch)/commitmeta.json` and drop the `find`.

Reply:
Make sense. I will change to use `builds/latest/$(arch)/commitmeta.json`.